src/main/java/com/three_tech_solutions/slot_app/controllers/interfaces/UserController.java

```diff
@@ -0,0 +1,17 @@
+package com.three_tech_solutions.slot_app.controllers.interfaces;
+
+import com.three_tech_solutions.slot_app.data.models.Student;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.List;
+import java.util.UUID;
+
+@RequestMapping("/users/{userId}/students")
+
+public interface UserController {
+    @GetMapping
+    List<Student> user(@PathVariable UUID userId);
```

Comment:
El ultimo comentario sobre esto, es que SIEMPRE los nombres de metodos y atributos, deberian ser representativos a lo que hacen.
En este caso el metodo es `user`, ¿pero qué significa `user`? El nombre del metodo puede dar a entender que estamos queriendo crear un usuario, obtener un usuario o actualizarlo. Cuando en realidad lo que queremos hacer es obtener los estudiantes de ese usuario, por lo que un nombre mas representativo seria

```suggestion
    List<StudentResponse> getUserStudents(@PathVariable UUID userId);
```